tests/nn/data_parallel/test_fsdp_regnet.py

```diff
@@ -343,8 +342,36 @@ def test1(temp_files, ddp_ref, precision, flatten):
     fsdp_config["mixed_precision"] = precision == "mixed"
     fsdp_config["flatten_parameters"] = flatten == "flatten"
 
-    if fsdp_config["mixed_precision"] and torch_cuda_version() < (11, 0):
-        pytest.skip("Only CUDA 11 is supported with AMP equivalency")
+    if fsdp_config["mixed_precision"]:
+        # For convolution layers, we need fp32_compute_dtype. Otherwise, we run
+        # into conv layer errors like below with mixed precision with larger
+        # regnet128Gf models when flatten_parameter is true:
+        #
+        #   RuntimeError: Unable to find a valid cuDNN algorithm to run convolution
+        #   or
+        #   RuntimeError: cuDNN error: CUDNN_STATUS_BAD_PARAM
+        #
+        # If you know the solutions to the above issue, please let me (min-xu-ai) know. Thanks!
```

Comment:
Can we post in some internal forum about this? Maybe https://fb.workplace.com/groups/1405155842844877/

Reply:
Good idea! Will do.

Reply:
done: https://fb.workplace.com/groups/1405155842844877/permalink/4614727588554337/